datadog/client.go

```diff
@@ -21,6 +21,13 @@ const (
 	// datadog.
 	DefaultBufferSize = 1024
 
+	// DefaultFilter is the default tag to filter before sending to
+	// datadog. Using the request path as a tag can overwhelm datadog's
+	// servers if there are too many unique routes due to unique IDs being a
+	// part of the path. Only change the default filter if there is a static
+	// number of routes.
+	DefaultFilter = "http_req_path"
```

Comment:
How about making this a slice:
```go
var (
    DefaultFilters = []string{"http_req_path"}
)
```
That would let programs potentially overwrite it, but also would be consistent to have the default be of the same type as the `Filters` field.

Reply:
Ah okay nice. I saw I couldn't add slices/arrays as constants, which is why I did it the way I did. Much better!

Reply:
Yep you can't make it a constant, but you can make it a top-level variable, this is common practice.